gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -1105,7 +1105,11 @@ String trimToPrefixWithoutGlob(String path) {
       if (e instanceof InterruptedException) {
         Thread.currentThread().interrupt();
       }
-      throw new IOException("Concurrent glob execution failed", e);
+      throw new IOException(
```

Comment:
How about we handle `ExecutionException` and `InterruptedException` separately ?

Reply:
Done.